apps/spark/src/test/java/com/linkedin/openhouse/jobs/spark/OperationsTest.java

```diff
@@ -89,9 +89,9 @@ public void testRetentionSparkAppWithStringPartitionColumns() throws Exception {
       rowValue.clear();
 
       rowValue.add("202%s-07-16-12");
-      // Rows with format different than the pattern provided, parsing fails silently for such
-      // values and date
-      // will not be deleted
+      // Rows with format different than the pattern provided. These rows will be deleted even
```

Comment:
Can we also add test that verifies that snapshot type after delete?

Reply:
Yes, added a test for it.